h/static/scripts/controllers/character-limit-controller.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+function CharacterLimitController(element) {
+  var counter = element.parentElement.querySelector(
+    '.js-character-limit-counter');
+  var textarea = element.querySelector('.js-character-limit-input');
```

Comment:
I'm using "input" here to refer to "whatever the input element is" be it a textarea or (later) an input or maybe in the future something else.